vars/rollingUpgradePipeline.groovy

```diff
@@ -156,6 +157,7 @@ def call(Map pipelineParams) {
                                                         dir('scylla-cluster-tests') {
                                                             def test_config = groovy.json.JsonOutput.toJson(pipelineParams.test_config)
                                                             def cloud_provider = getCloudProviderFromBackend(params.backend)
+                                                            // TODO: replace gce_image_db with the current scylla gce machine image
```

Comment:
seem you just need to remove the definition of `gce_image_db` from the rolling upgrades yamls, and it would kick in:
```python
            elif not self.get("gce_image_db") and self.get("cluster_backend") == "gce":
                if ":" in scylla_version:
                    gce_image = get_branched_gce_images(scylla_version=scylla_version)[0]
                else:
                    # gce_image.name format examples: scylla-4-3-6 or scylla-enterprise-2021-1-2
                    scylla_version_substr = f"scylla-{scylla_version.replace('.', '-')}"
                    for gce_image in get_scylla_gce_images_versions():
                        if gce_image.name.replace("-enterprise", "").startswith(scylla_version_substr):
                            break
                    else:
                        raise ValueError(f"GCE images for {scylla_version=} not found")
                self.log.debug("Found GCE image %s for scylla_version='%s'", gce_image.name, scylla_version)
````

Reply:
@KnifeyMoloko, I don't think running via the command inside the pipeline is the right way to switch into Scylla image as a base for those tests. small changes to configuration of the tests might be more reasonable.